tests/conftest.py

```diff
@@ -233,6 +233,21 @@ def xmatch_mock(xmatch_expected_dir):
     return pd.read_csv(os.path.join(xmatch_expected_dir, XMATCH_MOCK_FILE))
 
 
+@pytest.fixture
+def cone_search_expected_dir(test_data_dir):
+    return os.path.join(test_data_dir, "raw", "cone_search_expected")
+
+
+@pytest.fixture
+def cone_search_expected(cone_search_expected_dir):
+    return pd.read_parquet(os.path.join(cone_search_expected_dir, "catalog.parquet"))
```

Comment:
Could we make these CSV files instead, just for ease of human readability?  Or just a text file with the `source_id` field.